presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -151,7 +151,7 @@ public void setUp()
                 new SimpleTtlNodeSelectorConfig());
         PartitioningProviderManager partitioningProviderManager = new PartitioningProviderManager();
         nodePartitioningManager = new NodePartitioningManager(nodeScheduler, partitioningProviderManager, new NodeSelectionStats());
-        planFragmenter = new PlanFragmenter(metadata, nodePartitioningManager, new QueryManagerConfig(), new FeaturesConfig());
+        planFragmenter = new PlanFragmenter(metadata, nodePartitioningManager, new QueryManagerConfig(), new FeaturesConfig(), null);
```

Comment:
we shouldn't allow PlanCheckerProviderManager to be null (would break things in a real environment). Lets add a requireNonNull check to PlanFragmenter, and update teh tests to create an empty PlanCheckerProviderManager.

Reply:
Do we routinely use Mockito or EasyMock for stubs like this?

Reply:
No we don't use mocking in Presto.